sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeActionsAsyncClient.java

```diff
@@ -185,15 +195,37 @@ class AnalyzeActionsAsyncClient {
             EXTRACTIVE_SUMMARIZATION_TASKS, CUSTOM_ENTITY_RECOGNITION_TASKS, CUSTOM_SINGLE_CLASSIFICATION_TASKS,
             CUSTOM_MULTI_CLASSIFICATION_TASKS);
 
+
+    // Language Tasks
+    private static final String ABSTRACTIVE_SUMMARIZATION = "AbstractiveSummarization";
+    private static final String ENTITY_RECOGNITION = "EntityRecognition";
+    private static final String PII_ENTITY_RECOGNITION = "PiiEntityRecognition";
+    private static final String KEY_PHRASE_EXTRACTION = "KeyPhraseExtraction";
+    private static final String ENTITY_LINKING = "EntityLinking";
+    private static final String SENTIMENT_ANALYSIS = "SentimentAnalysis";
+    private static final String EXTRACTIVE_SUMMARIZATION = "ExtractiveSummarization";
+    private static final String HEALTHCARE = "Healthcare";
+    private static final String CUSTOM_ENTITY_RECOGNITION =  "CustomEntityRecognition";
+    private static final String CUSTOM_SINGLE_LABEL_CLASSIFICATION = "CustomSingleLabelClassification";
+    private static final String CUSTOM_MULTI_LABEL_CLASSIFICATION = "CustomMultiLabelClassification";
+
+    private static final String REGEX_ACTION_ERROR_TARGET_LANGUAGE_API =
+        String.format("#/tasks/(%s|%s|%s|%s|%s|%s|%s|%s|%s)/(\\d+)", ABSTRACTIVE_SUMMARIZATION,
```

Comment:
I don't know if it is the correct error target. Create an issue to track it: https://github.com/Azure/azure-sdk-for-java/issues/31852

Reply:
It looks like we got an update on what the error target format is in email

Reply:
So the error target format stays the same but only the taskName differs.
Legacy:  "#/tasks/entityRecognitionPiiTasks/0"
Language: "#/tasks/PiiEntityRecognition/0"

Reply:
@mssfang Language does not have the taskName in the error target. It's more like "#/tasks/items/0" per the email.

Reply:
My bad. Read the wrong one. Add a comment to remain self. https://github.com/Azure/azure-sdk-for-java/issues/31852#issuecomment-1298997608

It will be update in the next PR, dynamic classification PR. Thanks @kristapratico